AnkiDroid/src/main/java/com/ichi2/anki/reviewreminders/ReviewRemindersDatabase.kt

```diff
@@ -56,23 +56,23 @@ class ReviewRemindersDatabase {
     /**
      * Decode an encoded HashMap<[ReviewReminderId], [ReviewReminder]> JSON string.
      * @see Json.decodeFromString
-     * @throws SerializationException If the string is not a valid JSON string.
-     * @throws IllegalArgumentException If the decoded object is not a HashMap<[ReviewReminderId], [ReviewReminder]>.
+     * @throws SerializationException If the stored string is not a valid JSON string.
+     * @throws IllegalArgumentException If the decoded reminders map is not a HashMap<[ReviewReminderId], [ReviewReminder]>.
      */
     private fun decodeJson(jsonString: String): HashMap<ReviewReminderId, ReviewReminder> =
         Json.decodeFromString<HashMap<ReviewReminderId, ReviewReminder>>(jsonString)
 
     /**
      * Encode a Map<[ReviewReminderId], [ReviewReminder]> as a JSON string.
      * @see Json.encodeToString
-     * @throws SerializationException If the string is not a valid JSON string.
+     * @throws SerializationException If the stored string is somehow not a valid JSON string, even though the input parameter is type-checked.
      */
     private fun encodeJson(reminders: Map<ReviewReminderId, ReviewReminder>): String = Json.encodeToString(reminders)
 
     /**
      * Get the [ReviewReminder]s for a specific key.
-     * @throws SerializationException If the string is not a valid JSON string.
-     * @throws IllegalArgumentException If the decoded object is not a HashMap<[ReviewReminderId], [ReviewReminder]>.
+     * @throws SerializationException If the value associated to this key is not a string encoding valid JSON.
```

Comment:
nitpick/implemeter's choice 

```suggestion
     * @throws SerializationException If the value associated with this key is not valid JSON string.
```

Reply:
@ericli3690 Your call, ping me once you've seen this and let's get this merged